src/app/fdctl/run/tiles/fd_poh.c

```diff
@@ -569,8 +569,8 @@ fd_ext_poh_initialize( ulong         hashcnt_duration_ns, /* See clock comments
     /* Low power producer, maximum of one microblock per tick in the slot */
     ctx->max_microblocks_per_slot = ctx->ticks_per_slot;
   } else {
-    ctx->max_microblocks_per_slot = FD_PACK_MAX_MICROBLOCKS_PER_BLOCK;
-    FD_TEST( ctx->hashcnt_per_tick>=FD_PACK_MAX_MICROBLOCKS_PER_BLOCK );
+    /* We can set this to more or less whatever we want. */
```

Comment:
It no longer has any impact on shreds. It's just how many hashcnt we want to reserve.